jobs/kpi-forecasting/kpi_forecasting/models/base_forecast.py

```diff
@@ -40,7 +37,7 @@ class BaseForecast(abc.ABC):
     start_date: str
     end_date: str
     metric_hub: MetricHub
-    number_of_simulations: int = 1000
+    predict_historical_dates: bool = False
```

Comment:
Need to add a description of this arg to the docstring

Reply:
Fixed